.shippable.yml

```diff
@@ -29,6 +29,7 @@ build:
         options: "-e HOME=/home/buildslave --privileged=true --tty --net=bridge --user buildslave"
 
     ci:
+      - sudo apt-get install -y libasan1
```

Comment:
You're going to need to update the Zephyr documentation for various platforms to install this package